python/tests/integration/arcticdb/test_arctic.py

```diff
@@ -909,14 +919,6 @@ def test_get_uri(fixture, request):
     assert ac.get_uri() == storage_fixture.arctic_uri
 
 
-@AZURE_TESTS_MARK  # GH issue #1060
```

Comment:
Please could you explain why you are removing this.

Reply:
It has been merged into `test_s3_ssl_verification`. Bad naming. Will work on it